README.md

```diff
@@ -242,7 +242,7 @@ Refer  below for next steps when any indicator reports a `false`.
 | S0001        | Determines if the puppet service is running on agents.                                      | Starts the puppet service - `puppet resource service puppet ensure=running`                                                                                                                                                                                                                                                                                                                                                                                                    | If the service fails to start, open a Support ticket referencing S0001, and provide `syslog` and any errors output when attempting to restart the service.                                           |
-| S0002        | Determines if the pxp-agent service is running.                                         | Starts the pxp-agent service - `puppet resource service pxp-agent ensure=running`                                                                                                                                                                                                                                                                                                                                                                                                    | If the service fails to start, open a Support ticket referencing S0002, provide `syslog` any errors output when attempting to restart the service, and `/var/log/puppetlabs/pxp-agent/pxp-agent.log` |
+| S0002        | Determines if the pxp-agent service is running.                                         | Starts the pxp-agent service - `puppet resource service pxp-agent ensure=running`                                                                                                                                                                                                                                                                                                                                                                                                    | See [Documentation](https://portal.perforce.com/s/article/7606830611223)  |
```

Comment:
You removed the section for "what to send to support" rather than updating the self service section